### PR TITLE
Handling titles with year/country in legendastv provider.

### LINKED ISSUE #701: Series with year doesn't match in legendastv provider
For the latest subliminal: 2.0.5

`subliminal --debug --legendastv user pass download -v -l pt-BR -p legendastv -m 100 "Bull.2016.S01E01.HDTV.x264-LOL[ettv].mkv"`

```
Collecting videos
DEBUG:subliminal.cli:Collecting path Bull.2016.S01E01.HDTV.x264-LOL[ettv].mkv
DEBUG:subliminal.core:Found subtitles {}
INFO:subliminal.core:Refining video with metadata
INFO:subliminal.core:Refining video with tvdb
INFO:subliminal.refiners.tvdb:Searching series 'Bull'
DEBUG:subliminal.refiners.tvdb:Found 14 results
DEBUG:subliminal.refiners.tvdb:Discarding series u'Bull' mismatch on year 2000
DEBUG:subliminal.refiners.tvdb:Discarding series u'Bull Doctor' mismatch on year 2011
DEBUG:subliminal.refiners.tvdb:Discarding series u'Bull Fighting' mismatch on year 2007
DEBUG:subliminal.refiners.tvdb:Discarding series u'Bull (2015)' mismatch on year 2015
DEBUG:subliminal.refiners.tvdb:Found exact match on series u'Bull (2016)'
DEBUG:subliminal.refiners.tvdb:Discarding series u'Red Bull BC One' mismatch on year 2004
DEBUG:subliminal.refiners.tvdb:Discarding series u'Launched at Red Bull Studios' mismatch on year 2012
DEBUG:subliminal.refiners.tvdb:Discarding series u'Red Bull Mountainbike Chronicles' mismatch on year 2011
DEBUG:subliminal.refiners.tvdb:Discarding series u'Red Bull Ultimate Rush' mismatch on year 2011
DEBUG:subliminal.refiners.tvdb:Discarding series u'Mad Bull 34' mismatch on year 1990
DEBUG:subliminal.refiners.tvdb:Discarding series u'Red Bull Signature Series' mismatch on year 2012
DEBUG:subliminal.refiners.tvdb:Discarding series u"Ty Murray's Celebrity Bull Riding Challenge" mismatch on year 2007
DEBUG:subliminal.refiners.tvdb:Found series {u'seriesId': u'', u'rating': u'TV-14', u'overview': u'Dr. Jason Bull is the charming and cocky founder of successful trial consulting firm Trial Analysis Corporation, where he and his team of experts employ psychology, human intuition and high-tech data to understand jurors, lawyers, witnesses and defendants, and construct effective narratives to help their clients win.', u'firstAired': u'2016-09-20', u'zap2itId': u'EP02433697', u'aliases': [], u'networkId': u'', u'network': u'CBS', u'id': 311945, u'seriesName': u'Bull (2016)', u'status': u'Continuing', u'added': u'2016-05-19 01:15:35', u'imdbId': u'tt5827228', u'lastUpdated': 1476000557, u'addedBy': 396427, u'genre': [u'Drama'], u'banner': u'graphical/311945-g.jpg', u'airsTime': u'9:00 pm', u'siteRatingCount': 5, u'airsDayOfWeek': u'Tuesday', u'runtime': u'45', u'siteRating': 7.6}
INFO:subliminal.refiners.tvdb:Getting series episode 1x1
DEBUG:subliminal.refiners.tvdb:Found episode {u'seriesId': 311945, u'airsAfterSeason': None, u'dvdEpisodeNumber': 1, u'productionCode': u'101', u'overview': u'Dr. Jason Bull, the brilliant, brash founder of a prolific trial consulting firm, combines psychology, human intuition and high tech data to create winning strategies that steer high-stakes trials in his clients\u2019 favor.', u'airedSeasonID': 671546, u'thumbAdded': u'2016-09-20 18:42:00', u'episodeName': u'The Necklace', u'airedSeason': 1, u'airsBeforeSeason': None, u'showUrl': u'', u'id': 5654901, u'dvdDiscid': u'', u'filename': u'episodes/311945/5654901.jpg', u'dvdSeason': 1, u'writers': [u'Paul Attanasio', u'Phil McGraw'], u'airedEpisodeNumber': 1, u'absoluteNumber': 1, u'thumbHeight': u'225', u'imdbId': u'tt5692502', u'lastUpdated': 1475759426, u'director': u'Rodrigo Garcia', u'lastUpdatedBy': 420157, u'dvdChapter': None, u'airsBeforeEpisode': None, u'language': {u'overview': u'en', u'episodeName': u'en'}, u'thumbAuthor': 360202, u'siteRatingCount': 3, u'directors': [u'Rodrigo Garcia'], u'firstAired': u'2016-09-20', u'guestStars': [], u'siteRating': 8.3, u'thumbWidth': u'400'}
INFO:subliminal.core:Refining video with omdb
DEBUG:subliminal.refiners.omdb:No need to search
1 video collected / 0 video ignored / 0 error
Downloading subtitles
INFO:subliminal.core:Listing subtitles with provider 'legendastv' and languages set([<Language [pt-BR]>])
INFO:subliminal.core:Initializing provider legendastv
INFO:subliminal.providers.legendastv:Logging in
DEBUG:subliminal.providers.legendastv:Logged in
INFO:subliminal.providers.legendastv:Searching title u'bull'
WARNING:subliminal.providers.legendastv:No season detected for title 32540
DEBUG:subliminal.providers.legendastv:Found 30 titles
DEBUG:subliminal.core:Terminating initialized providers
INFO:subliminal.core:Terminating provider legendastv
INFO:subliminal.providers.legendastv:Logging out
DEBUG:subliminal.providers.legendastv:Logged out
0 subtitle downloaded for Bull.2016.S01E01.HDTV.x264-LOL[ettv].mkv
```

There's no series name match because in legendas.tv website the tv show is named: Bull 2016
http://legendas.tv/legenda/sugestao/bull%202016 :  `"dsc_nome":"Bull (2016)"`

I believe that the correct fix is to use the [same idea](https://github.com/Diaoul/subliminal/blob/master/subliminal/refiners/tvdb.py#L298) from tvdb refiner and improve this [logic](https://github.com/Diaoul/subliminal/blob/master/subliminal/providers/legendastv.py#L226-L248)

When I get some time, I could make a PR for it

### PR DESCRIPTION
**Pull request by ratoaq2**
Series titles with year (and country) should be parsed correctly.
The example is: `"dsc_nome":"Bull (2016)",`
http://legendas.tv/legenda/sugestao/bull

```
   {  
      "_index":"filmes",
      "_type":"filme",
      "_id":"42047",
      "_score":null,
      "_source":{  
         "id_filme":"42047",
         "id_imdb":"5827228",
         "tipo":"S",
         "int_genero":"0",
         "dsc_imagen":"legendas_tv_20160922004714.jpg",
         "dsc_nome":"Bull (2016)",
         "dsc_sinopse":"....",
         "dsc_data_lancamento":"2016",
         "dsc_url_imdb":"http:\/\/www.imdb.com\/title\/tt5827228\/",
         "dsc_nome_br":"Bull (2016) - 1\u00aa Temporada",
         "soundex":null,
         "temporada":"1",
         "id_usuario":"3318434",
         "flg_liberado":"0",
         "dsc_data_liberacao":null,
         "dsc_data":"2016-09-22T00:48:04",
         "dsc_metaphone_us":null,
         "dsc_metaphone_br":null,
         "episodios":null,
         "flg_seriado":null,
         "last_used":"0",
         "deleted":false
      },
```

This fixes #701

### CODE REVIEW

**Comment by fernandog** [comment]
@Diaoul can you check this please?

**Comment by fernandog** [comment]
Also fixes https://github.com/Diaoul/subliminal/issues/693

File: tests/test_legendastv.py [hunk 1]
```diff
@@ -165,6 +165,16 @@ def test_search_titles_without_season_information():
 
 @pytest.mark.integration
 @vcr.use_cassette
+def test_search_titles_containing_year_information():
+    with LegendasTVProvider() as provider:
+        titles = provider.search_titles('Bull')
+    assert 42047 in titles.keys()
+    t = titles[42047]
+    assert (t['title'], t['year']) == ('Bull', 2016)
```
Comment (Diaoul) [inline tests/test_legendastv.py]: You shouldn't need the explicit parenthesis.

**Comment by ratoaq2** [comment]
Rebased and removed the parenthesis

**Pull request merged**